### PR TITLE
Take into account first week day on Mac OS X user preferences

### PR DESCRIPTION
**Pull request by andresgutgon**
### What is this?
This PR is trying to address [this issue](https://github.com/mike-engel/Barnacal/issues/3)

More info there.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -137,6 +142,52 @@ const showAbout = aboutWindow => () => {
...
+    'AppleFirstWeekday',
+    'dictionary'
+  );
+
+  if (!Object.keys(firstWeekdayPref).length) return null
+
+  // key in this example is `gregorian` and day is `2`
+  // { gregorian = 2; }
+  const key = Object.keys(firstWeekdayPref)[0]
+  return firstWeekdayPref[key]
```
Comment (mike-engel) [inline index.js]: Can this just be `return firstWeekdayPref.gregorian`, or does this change if the user doesn't use a gregorian calendar?

File: index.js [hunk 2]
```diff
@@ -166,6 +217,10 @@ const configureWindow = () => {
 
   window.loadURL(htmlPath);
 
+
+  // Debug on Chrome Dev Tools
+  // window.webContents.openDevTools()
```
Comment (mike-engel) [inline index.js]: This should be removed

File: index.js (hunk 1 above)
Comment (andresgutgon) [reply index.js]: This way we don't care for the key, we just want the value which the number of the first day of week

File: index.js (hunk 2 above)
Comment (andresgutgon) [reply index.js]: Not having played with Electron previously it took me a while to start debugging. I think it doesn't hurt for the next collaborator to see this hint

Wht do you think?

I can also add a comment in the README instead

Comment (mike-engel) [reply index.js]: Ah, I see. Yeah, I think this would be better suited in the README

**Comment by andresgutgon** [comment]
Let me know if I can do anything else or if you see it ok :)

**Comment by mike-engel** [comment]
Sorry for the delay @andresgutgon, but this has now been released as 0.8.3. Thanks again!

**Comment by andresgutgon** [comment]
Thanks @mike-engel 👏

**Pull request merged**